.github/workflows/nuget.yml

```diff
@@ -0,0 +1,39 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Nuget
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+        with:
+          ref: 'v0.6.0'
+      
+      # Deploy NuGet so we can call the pack and push cmds
+      - name: Setup NuGet.exe for use with actions
+        uses: NuGet/setup-nuget@v1.0.7
+        with:
+          nuget-api-key: ${{ secrets.nuget_api_key }}
```

Comment:
Do we need access to the Nuget package to make an API token? I made an account with the same user name if you don't mind :)

Reply:
Yes, he needs to add one of us (or ideally both) as a owner for the package and we then need to create a secret [here](https://github.com/Thalhammer/jwt-cpp/settings/secrets/actions) with the push token.

Reply:
You are the only one who can add secrets AFAIK, I dont have access to the settings page where they can be added

(my evil genius plan to steal power is working muahahah 😈 )

Reply:
> I dont have access to the settings page where they can be added

Guess I need to add them myself then :smiling_imp:

Reply:
Added you both access as owners directly via NuGet.org so you can manage the package even before we publish the next one:
![image](https://user-images.githubusercontent.com/19294348/195988805-12d80e6e-7ff9-49df-90df-ee122bf6bcf7.png)
And the next commit will have you both added as owners in the .nuspec file, together with the version updated to a test one so you can experiment with running the workflow with your own token :)